code/fmriprep/submit-fmriprep.sh

```diff
@@ -38,7 +38,7 @@
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
 
-DATADIR="/oak/stanford/groups/russpold/inprocess/cprovins/hcph-pilot/"
+DATADIR=<workdir>/data/hcph-dataset
```

Comment:
You can use secrets and settings here:

```suggestion
DATADIR={{ secrets.data.curnagl_workdir | default('<workdir>') }}/data/hcph-dataset
```

You'll need to make sure you add/confirm the ``curnagl_workdir`` setting in the secrets yaml file.

Reply:
I did not want to use secrets here, because we do not render this code in the documentation.
We just point that it exists by telling the user to

> cd code/fmriprep
> bash submit-fmriprep.sh

Reply:
Let's render it, the mkdocs will add a copy icon, so it's way more convenient than having the script on a file.